### DISCUSSION TITLE
Deepsleep from the GS should not resume in MainUI

### DISCUSSION THREAD

**Discussion by Sichroteph** [OnionUI/Onion#594] [Ideas]
It should resume to the GS. It is to be discussed, but it also seems intuitive to resume to the game the GS tile was.

**Comment by Olywa** [comment]
I did not see this until now. I'm not sure if you knew but start-up behaviour is configurable in the Tweaks app. 

If you enter into deep sleep from in game then yes, you will resume to that game. If you enter deep sleep from elsewhere (MainUI) you can decide whether you boot to GS, MainUI or even RetroArch. These options are in Apps > Tweaks > System > Startup 

I guess the addition of a GS _in-game_ option would be good (instead of only GS _menu_).
